src/main/java/seedu/address/logic/Logic.java

```diff
@@ -28,10 +28,10 @@
      *
      * @see seedu.address.model.Model#getGradPad()
      */
-    ReadOnlyGradPad getAddressBook();
+    ReadOnlyGradPad getGradPad();
 
     /** Returns an unmodifiable view of the filtered list of persons */
-    ObservableList<Person> getFilteredPersonList();
+    ObservableList<Module> getFilteredModuleList();
 
     /**
      * Returns the user prefs' address book file path.
```

Comment:
```suggestion
     * Returns the user prefs' GradPad file path.
```
can try using replace tool to search for remaining address book